cluster/gce/util.sh

```diff
@@ -646,6 +577,7 @@ function kube-up {
   for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
     create-route "${MINION_NAMES[$i]}" "${MINION_IP_RANGES[$i]}" &
     add-instance-metadata "${MINION_NAMES[$i]}" "node-ip-range=${MINION_IP_RANGES[$i]}" &
+    add-instance-metadata "${MINION_NAMES[$i]}" "node-name=${MINION_NAMES[$i]}" &
```

Comment:
(the instance metadata update uses an opportunistic locking approach, so this just pounds the same metadata from two processes and it fails. It's easy to see from even a tiny node count.)